giddy-banana/app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
```

Comment:
Something like that?
```suggestion
import express from "express";
```

Reply:
Yes

Reply:
Done